docs/fides/docs/guides/policies.md

```diff
@@ -0,0 +1,112 @@
+    name: Employee Data Processing
+    description: Restrict Employee Data Processing to only what is required for systematic business functions.
+    rules:
+      - name: Disallow Non-System Use of Employee Data
+        data_categories:
+          matches: ANY # If any of these data categories are being used
+          values:
+            - account
+            - user
+        data_uses:
+          matches: OTHER # And the use of the data is something other than...
+          values:
+            - provide.system
+            - improve.system
+        data_subjects:
+          matches: ANY # And the data subject is an employee
+          values:
+            - employee
+        # And the data is identifiable, trigger a violation
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Respecting Biometric PII
+
+```yaml title="policy.yml"
+policy:
+  - fides_key: biometric_data_policy
+    name: Biometric Data Policy
+    description: Policy that describes valid uses of biometric and health data.
+    rules:
+      - name: Disallow Biometrics for Profit.
+        description: Disallow biometric data being used for profit-related use-cases.
+        data_categories:
+          matches: ANY # If any of these data categories are being used
+          values:
+            - user.derived.identifiable.biometric_health
+            - user.provided.identifiable.biometric
+        data_uses:
+          matches: ANY # And the use of the data is for any of the following...
+          values:
+            - advertising
+            - train_ai_system
+            - improve
+            - third_party_sharing
+        data_subjects:
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is identifiable, trigger a violation
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized.identified
+```
+
+## Anonymous Derived User Contact Data
+
+```yaml title="policy.yml"
+policy:
+  - fides_key: protect_derived_user_data
+    name: Protect Derived User Data
+    description: Policy that describes the proper use of derived user data.
+    rules:
+      - name: Disallow Non-Anonymized Derived User Data.
+        description: Require that any use of derived user data must be de-identified to the anonymous level, as opposed to the pseudonymous.
+        data_categories:
+          matches: ANY # If any of these data categories are being used
+          values:
+            - user.derived.identifiable.biometric_health
+        data_uses:
+          matches: NONE # And for any data use
+          values: []
+        data_subjects:
+          matches: ANY # And the data subject is a customer
+          values:
+            - customer
+        # And the data is either unlinked_pseudonymized or more identifiable, trigger a violation
+        data_qualifier: aggregated.anonymized.unlinked_pseudonymized
```

Comment:
The definition of "Unlinked pseudonymized data" is data for which all identifiers are erased or substituted by aliases for which the assignment function *is erased or irreversible, such that the linkage cannot be re-established by reasonable efforts of anyone including the party that performed them*.
I would be included to say that "unlinked pseudonymized" is an acceptable level of deidentifiability under the EDPB ruling. however, vanilla `pseudo` or `identified` is unacceptable, so maybe this should be `data_qualifier: aggregated.anonymized.unlinked_pseudonymized.pseudonymized`?